site/content/en/docs/user/kwokctl-snapshot/etcd-snapshot.demo

```diff
@@ -0,0 +1,29 @@
+# Let's getting started with kwokctl!
+kwokctl create cluster
+
+# Create an nodes.
```

Comment:
See this [review](https://github.com/kubernetes-sigs/kwok/pull/1100/files/466a82ab10ef6bd11947ecf10e34e46e34f9a6ba..5a335fc81c238667d5fbbc176b84642b5679b6a8#r1610727316) below.

```suggestion
# Create the nodes.
```